R/client.R

```diff
@@ -6,20 +6,21 @@
 #' @importFrom RProtoBuf P serialize read new
 #' @export
 grpc_client <- function(impl, channel) {
-  
+
+    timeout <- getOption("grpc.timeout")
```

Comment:
You can add a default here (5000), otherwise existing code will need to be updated.